### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by yepzdk**
Ticket: https://jira.itkdev.dk/browse/ODAA-108
* Basic styling
* Dashbord styling
* Login styling

### CODE REVIEW

File: assets/js/login.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require('../scss/login.scss')
+
+$('#login-form .form-control').on('blur input focus', function () {
+  var $field = $(this).closest('.form-group')
+  if (this.value) {
```
Comment (rimi-itk) [inline assets/js/login.js:6]: For consistent use of `jQuery` this could be changed to `$(this).val()`.

File: assets/js/login.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    $field.addClass('filled')
+  } else {
+    $field.removeClass('filled')
+  }
+})
+
+$('#login-form .form-control').on('focus', function () {
+  var $field = $(this).closest('.form-group')
+  if (this) {
```
Comment (rimi-itk) [inline assets/js/login.js]: Should this be `this.value` (or `$(this).val()`)?

File: assets/scss/login.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            color: $gray-300;
+            padding-top: $spacer;
+            padding-bottom: $spacer;
+        }
+        // input {
+        //     background: rgba($color: $black, $alpha: .5) !important;
+        // }
+        // label {
+        //     border: $gray-300 1px solid !important;
+        // }
```
Comment (rimi-itk) [inline assets/scss/login.scss]: This comment should be removed.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Looks good. A few comments should be addressed.

**Comment by yepzdk** [comment]
@rimi-itk I have added new commits to resolve your comments.

File: assets/js/login.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  if ($(this).val()) {
+    $field.addClass('filled')
+  } else {
+    $field.removeClass('filled')
+  }
+})
+
+$('#login-form .form-control').on('focus', function () {
+  var $field = $(this).closest('.form-group')
+  if ($(this)) {
```
Comment (rimi-itk) [inline assets/js/login.js]: `$(this)` is always true(ish) here – I think you mean `$(this).val()`.

File: assets/js/login.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    $field.addClass('filled')
+  } else {
+    $field.removeClass('filled')
+  }
+})
```
Comment (rimi-itk) [inline assets/js/login.js]: An event handler for the `focus` event (on the same elements) is set in both lines 3 and 12. Unless I'm missing something the second event handler is superfluous.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
I think we have some code duplication.

File: assets/js/login.js (hunk 5 above)
Comment (yepzdk) [reply assets/js/login.js]: New commit will address this.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**